src/client/App.css

```diff
@@ -1,42 +1,120 @@
+ul {
```

Comment:
I like how you wrote lots of CSS instead of finishing the challenge. You are a legend sir